kimchi/src/precomputed_srs.rs

```diff
@@ -33,6 +33,9 @@ where
     let file =
         File::open(srs_path.clone()).unwrap_or_else(|_| panic!("missing SRS file: {srs_path:?}"));
     let reader = BufReader::new(file);
+    // Note: In tests, this read takes significant amount of time (2 min) due
```

Comment:
This isn't useful: the only time that you'd be here and need this observation is if you were investigating it, and there's no practical solution beyond what's in `CONTRIBUTING.md` anyway.

Also, orthogonal to this PR.

Delete.